webapp/src/app/routes/json/json.component.ts

```diff
@@ -22,14 +22,10 @@ export class JsonComponent {
   fromJSON(): void {
     const blockMap: BlockDataMap = {};
     try {
+      // BlockData from JSON.parse doesn't have any method
```

Comment:
.. because Typescript is weird like that.

:-)

Reply:
It happens, because we run not typescript, but javascript in a browser. JS hasn't our custom types, so we don't have any problem here.
We don't get error in IDE, because IDE doesn't know, which type is going to be returned from JSON.parse.
And JSON.parse can't return object with methods, because JSON has no function type.

Reply:
https://i.imgur.com/D8W8GkE.png